apps/etterlatte-pdltjenester/.nais/dev.yaml

```diff
@@ -56,6 +56,4 @@ spec:
         - application: etterlatte-opplysninger-fra-pdl
         - application: etterlatte-gyldig-soeknad
         - application: etterlatte-hendelser-pdl
-        - application: etterlatte-behandling
-        - application: etterlatte-saksbehandling-ui-lokal
```

Comment:
Er ui-lokal nødvendig for å kjøre mot dev fra lokalt?

Reply:
Jepp